server/sandbox_run.go

```diff
@@ -405,8 +406,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 
 	// setup the network
 	if !hostNetwork {
-		podNamespace := ""
-		if err = s.netPlugin.SetUpPod(netNsPath, podNamespace, id, containerName); err != nil {
+		if err = s.netPlugin.SetUpPod(netNsPath, namespace, name, id); err != nil {
```

Comment:
this `name` here is wrong and it's causing the failure in https://github.com/kubernetes-incubator/cri-o/pull/489#issuecomment-299449015

this `name` here is `namespace + name + attempt` and openshift-sdn fails to lookup the pod with that name. Instead, this patch works just fine if you use the original name from `req.GetConfig().GetMetadata().Name` (which may be still an error to use `Name` directly but then there's a bug in CNI code I guess or kube).